src/library_browser.js

```diff
@@ -1381,6 +1381,36 @@ var LibraryBrowser = {
     exit(status);
   },
 
+  emscripten_get_window_title__proxy: 'sync',
+  emscripten_get_window_title__sig: 'iv',
+  emscripten_get_window_title: function() {
+    var buflen = 256;
+
+    if (!_emscripten_get_window_title.buffer) {
+      _emscripten_get_window_title.buffer = _malloc(buflen);
+    }
+
+    writeAsciiToMemory(
+      document.title.slice(0, buflen - 1),
+      _emscripten_get_window_title.buffer
+    );
+
+    return _emscripten_get_window_title.buffer;
+  },
+
+  emscripten_set_window_title__proxy: 'sync',
+  emscripten_set_window_title__sig: 'vi',
+  emscripten_set_window_title: function(title) {
+    setWindowTitle(AsciiToString(title));
+  },
+
+  emscripten_get_screen_size__proxy: 'sync',
+  emscripten_get_screen_size__sig: 'vii',
+  emscripten_get_screen_size: function(width, height) {
+    if (width) {{{ makeSetValue('width', '0', 'screen.width', 'i32') }}};
+    if (height) {{{ makeSetValue('height', '0', 'screen.height', 'i32') }}};
```

Comment:
is it worth making these optional? we'd need to document that if we want it. but it seems simpler to assume the caller wants to receive both of them?

Reply:
I think I'd also tend to agree that it's simpler to assume the caller wants both the width and height. This would be used specifically in the SDL2 port, in `Emscripten_VideoInit`[1], which gets the width and height in a single call. If the caller only wants one or the other, they can simply pass NULL for either parameter, and that value will be ignored.

https://github.com/jakogut/SDL2/blob/2b641edaad57e32a709a70a1609749ac26b6ec0c/src/video/emscripten/SDL_emscriptenvideo.c#L146

Reply:
Sorry for the delay - I missed this email somehow.

Sounds like this will be used where both width and height are used, so let's not make it optional? It just increases code size, requires new docs, and overall doesn't seem worth it.

I think the PR can land, once we fix that and there's a small typo comment from earlier! Thanks for the work here.